graph/resolver/post.resolvers.go

```diff
@@ -107,3 +108,13 @@ func (r *postResolver) ImportedContent(ctx context.Context, obj *model.Post) (*m
 func (r *Resolver) Post() generated.PostResolver { return &postResolver{r} }
 
 type postResolver struct{ *Resolver }
+
+// !!! WARNING !!!
+// The code below was going to be deleted when updating resolvers. It has been copied here so you have
+// one last chance to move it out of harms way if you want. There are two reasons this happens:
+//  - When renaming or deleting a resolver the old code will be put in here. You can safely delete
+//    it when you're done.
+//  - You have helper methods in this file. Move them out to keep these resolver files clean.
+func (r *postResolver) PostType(ctx context.Context, obj *model.Post) (model.PostType, error) {
```

Comment:
Do we want to delete this?

Reply:
didn't push up the commit that had it deleted. good catch